### PR TITLE
axoloti: init at 1.0.12

### PR DESCRIPTION
**Pull request by TealG**
###### Things done

- [ ] Tested using sandboxing
  ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS,
    or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file)
    on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] Linux
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

**Comment by TealG** [comment]
Fixed some typos, now tests pass.

**Comment by Mic92** [comment]
Why I am the author of this commit :)

File: pkgs/applications/networking/browsers/palemoon/default.nix [hunk 1]
```diff
@@ -5,19 +5,19 @@
 , gtk2, hunspell, icu, libevent, libjpeg, libnotify
 , libstartup_notification, libvpx, makeWrapper, mesa
 , nspr, nss, pango, perl, python, libpulseaudio, sqlite
-, unzip, xlibs, which, yasm, zip, zlib
+, unzip, xlibs, which, yasm, zip, zlib, gcc
```
Comment (Mic92) [inline pkgs/applications/networking/browsers/palemoon/default.nix]: Please make this a separate commit.

File: pkgs/applications/misc/octoprint/default.nix [hunk 2]
```diff
@@ -60,7 +60,7 @@ in pythonPackages.buildPythonApplication rec {
     owner = "foosel";
     repo = "OctoPrint";
     rev = version;
-    sha256 = "06l8khbq3waaaa4cqpv6056w1ziylkfgzlb28v30i1h234rlkknq";
+    sha256 = "1hci8cfmbzcghla1vmrcn6zicm8nj50drm7gp2hkr0drglq5fgr2";
```
Comment (Mic92) [inline pkgs/applications/misc/octoprint/default.nix]: Please make this a separate commit.

File: pkgs/applications/audio/google-play-music-desktop-player/default.nix [hunk 3]
```diff
@@ -4,7 +4,7 @@
 }:
 
 let
-  version = "4.3.0";
+  version = "4.4.0";
```
Comment (Mic92) [inline pkgs/applications/audio/google-play-music-desktop-player/default.nix]: Please make this a separate commit.

File: pkgs/applications/audio/axoloti/default.nix [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+    faketime "1970-01-01 00:00:00" ant -Dbuild.runtime=true
+  '';
+
+  installPhase = ''
+    mkdir -p $out/bin $out/share/axoloti
+
+    cp -r doc firmware chibios platform_linux CMSIS *.txt $out/share/axoloti/
+    install -vD dist/Axoloti.jar $out/share/axoloti/
+
+    cat > $out/bin/axoloti <<EOF
```
Comment (Mic92) [inline pkgs/applications/audio/axoloti/default.nix]: makeWrapper is preferred here, see https://nixos.org/nixpkgs/manual/#sec-language-java as an example

**Review by Mic92 — APPROVED** [review]
The package itself looks good to me.

File: pkgs/applications/networking/browsers/palemoon/default.nix (hunk 1 above)
Comment (TealG) [reply pkgs/applications/networking/browsers/palemoon/default.nix]: indeed, this does not belong there, must be a merge issue since I didn't modify this myself -- will check

**Comment by joachifm** [comment]
@TealG are you still working on this?

**Comment by TealG** [comment]
@joachifm should be done now, fixed merge issues + replaced cat with makeWrapper as @Mic92 suggested

File: lib/maintainers.nix [hunk 5]
```diff
@@ -590,6 +590,7 @@
   taku0 = "Takuo Yonezawa <mxxouy6x3m_github@tatapa.org>";
   tari = "Peter Marheine <peter@taricorp.net>";
   tavyc = "Octavian Cerna <octavian.cerna@gmail.com>";
+  TealG = "Teal Gaure <~@Teal.Gr>"
```
Comment (Mic92) [inline lib/maintainers.nix]: missing leading semicolon.

**Comment by TealG** [comment]
@Mic92 fixed

**Comment by Mic92** [comment]
Did you close the pr by accident?

**Comment by TealG** [comment]
@Mic92 yes got closed by a push accidentally.. created a new one at #30086
